src/styles/components/breadcrumbs.scss

```diff
@@ -25,6 +25,54 @@
 
 @media screen and (max-width: 1024px) {
 	.breadcrumbs {
-		display: none;
+		display: inline-block;
+		text-align: left;
+		margin: auto 0;
+		h4 {
+			font-size: 20px;
+			display: inline-block;
+			text-align: left;
+		}
+		nav {
+			display: inline-block;
+		}
+	}
+}
+
+@media screen and (max-width: 768px) {
+	.breadcrumbs {
+		margin: auto 0 0;
+	}
+}
+
+@media screen and (max-width: 768px) {
+	.breadcrumbs {
+		display: inline-block;
+		text-align: left;
+		//position: absolute;
```

Comment:
@Aringoaway Can you pls remove those commented lines? If there is no `TODO:` written, I suppose those comments are redundant and it can be removed, right?